pkg/standalone/standalone.go

```diff
@@ -415,19 +417,20 @@ func runPlacementService(wg *sync.WaitGroup, errorChan chan<- error, info initIn
 
 	placementContainerName := utils.CreateContainerName(DaprPlacementContainerName, info.dockerNetwork)
 
-	image, err := resolveImageURI(daprImageInfo{
+	imgInfo := daprImageInfo{
 		ghcrImageName:      daprGhcrImageName,
 		dockerHubImageName: daprDockerImageName,
 		imageRegistryURL:   info.imageRegistryURL,
 		imageRegistryName:  defaultImageRegistryName,
-	})
+	}
+	image, err := resolveImageURI(imgInfo)
 	if err != nil {
 		errorChan <- err
 		return
 	}
 	image = getPlacementImageWithTag(image, info.runtimeVersion)
 
-	if defaultImageRegistryName == githubContainerRegistryName {
+	if checkFallbackImg(imgInfo, info.fromDir) {
 		if !TryPullImage(image) {
 			print.InfoStatusEvent(os.Stdout, "Placement image not found in Github container registry, pulling it from Docker Hub")
 			image = getPlacementImageWithTag(daprDockerImageName, info.runtimeVersion)
```

Comment:
Seems only placement image has a fallback. If that is the case, could you please update method comments to reflect the intent, so that a future developer is quickly able to get context.

Reply:
I have updated the method name to more idea about the use

Reply:
Can we rename it to convey what it's doing? Something like `if useGitHubRegistry(imgInfo, info.fromDir)`.

Reply:
I have updated this to `checkFallbackImgForPlacement`.Basically this function is just checking whethere this `init` flow qualifies for `tryPull` or not

Reply:
With the `check` prefix in `checkFallbackImgForPlacement`, it might not be very apparent on (1) what the function does and (2) what's the consumer's intent. If this returns true, we "check" fallback image for placement service - is slightly confusing. Instead, if it says "pullFromGithubRegistry", or "useGitHubRegistry", it might both justify what the method is doing, and how the consumer is using it. Thoughts?